deploy-oke-using-ubuntu/cli/env.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+export COMPARTMENT_ID="<COMPARTMENT_ID>" # MUST BE REPLACED
+COMPARTMENT_NAME=$(oci iam compartment get --compartment-id "$COMPARTMENT_ID" --query "data.name" --raw-output)
+export COMPARTMENT_NAME
+export REGION_NAME="<REGION_NAME>" # MUST BE REPLACED
+export VCN_NAME="oke-custom-vcn"
+
+# Used for security list rules
+export SERVICE_GATEWAY_NAME="oke-service-gateway"
+export NAT_GATEWAY_NAME="oke-nat-gateway"
+export CIDR_BLOCK_VCN="10.0.0.0/16"
+export CIDR_BLOCK_SUBNET_NODES="10.0.10.0/24"
+export CIDR_BLOCK_SUBNET_CP="10.0.0.0/28"
+export CIDR_BLOCK_SUBNET_SVCLB="10.0.20.0/24"
+export IPPROTO_ALL="all"
+export IPPROTO_ICMP="1"
+export IPPROTO_TCP="6"
+export ROUTE_TABLE_NAME="oke-public-route-table"
+export NODES_SECLIST_NAME="oke-nodes-seclist"
+export CP_SECLIST_NAME="oke-control-plane-seclist"
+export SVCLB_SECLIST_NAME="oke-service-lb-seclist"
+
+# Used for subnet configuration
+export NODES_SUBNET_NAME="oke-nodes-subnet"
+export CONTROL_PLANE_SUBNET_NAME="oke-control-plane-subnet"
+export SERVICE_LOAD_BALANCER_NAME="oke-service-lb-subnet"
+
+# Cluster configuration
+export CLUSTER_NAME="oke-cluster"
+export KUBE_CONFIG_PATH="$HOME/.kube/config"
+export KUBE_VERSION="v1.32.1"
+export CNI_TYPE="FLANNEL_OVERLAY"
+export IMAGE_OCID="ocid1.image.oc1.phx.aaaaaaaahacoznvu7ozscecpk4wi3gvqwu43ot5itbuwrj7yu3eei2xpbloq"
```

Comment:
Remove this image and force the users to supply their own. May be worth pointing them to the docs page on how to register an Ubuntu OKE specific image too.